org.deidentifier.arx.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: org.deidentifier.arx
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+
+command: arx
+finish-args:
+  - --socket=x11
+#  - --device=dri
+  - --share=ipc
+  # network: the app can connect to DBMS
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk17/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: arx-binary
+    buildsystem: simple
+    build-commands:
+        - install -D -ma+rx arx.sh /app/bin/arx
+        - install -D -ma+r arx-gtk-64.jar /app/bin/arx-gtk-64.jar
```

Comment:
The JAr is executed by the script and not directly, So you may want to put it in another folder than bin

Reply:
yeah, good point. I don't care where that file lives. Do you have suggestions? lib?

Reply:
Maybe /app/arx-gtk